packages/@aws-cdk/aws-logs/lib/cross-account-destination.ts

```diff
@@ -6,6 +6,24 @@ import { CfnDestination } from './logs.generated';
 import { ILogSubscriptionDestination, LogSubscriptionDestinationConfig } from './subscription-filter';
 import { ArnFormat } from '@aws-cdk/core';
 
+abstract class CrossAccountDestinationBase extends cdk.Resource implements ILogSubscriptionDestination {
```

Comment:
Why are we adding this base class? I know this is a common pattern elsewhere, but this base class doesn't add much value...are there multiple other types of `CrossAccountDestination` that would actually inherit from this?